app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -196,6 +197,25 @@ def enabled_repos
       respond_for_show :resource => result
     end
 
+    api :PUT, "/systems/:id/deb_package_profile", N_("Update installed deb packages")
+    param :deb_package_profile, Hash, :required => true do
+      param :deb_packages, Array, :required => true do
+        param :name, String, :required => true
+        param :architecture, String, :required => true
+        param :version, String, :required => true
+      end
+    end
+    param :id, String, :desc => N_("UUID of the system"), :required => true
+    def deb_package_profile
```

Comment:
Is the reason we have 'deb_package_profile' because the format is different than the RPM package profile?

Reply:
Yes, i guess, that was the reason.